### PR TITLE
#195 Used cache in the API calls

### LINKED ISSUE #195: Add big cache to appx api endpoint for purchases
Add a 1 day cache here so dependency on this api reduces - https://github.com/code100x/cms/blob/main/src/utiles/appx.ts#L61

### PR DESCRIPTION
**Pull request by adexh**
Fixes #195
## Changes

- Added use of the Cache class in the purchase API

### CODE REVIEW

File: src/utiles/appx.ts [hunk 1]
```diff
@@ -91,5 +101,6 @@ export async function getPurchases(email: string) {
       }
     }
   }
+  Cache.getInstance().set('courses', [email], responses, 60 * 60 * 24);
```
Comment (hkirat) [inline src/utiles/appx.ts:104]: only this line and the cache get for this is needed
No need for any LOCAL_CMS_PROVIDER logic that can remain uncached

**Review by hkirat — CHANGES_REQUESTED** [review]
just one comment

**Comment by adexh** [comment]
@hkirat Done Sir.
Removed the logic for LOCAL_CMS_PROVIDER and kept its response uncached.

**Comment by hkirat** [comment]
Thankyou! 
/bounty $15

**Pull request merged**